TENETS.md

```diff
@@ -0,0 +1,58 @@
+1. [Be kind and professional](#be-kind-and-professional)
+
+## Break large changes into manageable chunks
+Every contributor is expected to limit the size of their pull requests to something that can be considered reasonable to review. Excessively large changes are difficult to review and often hide bugs and less than ideal architectures.
+
+## Semantic versioning
+MRTK3 packages are individually versioned and follow semantic versioning rules. Please see [Semantic Versioning 2.0.0](https://semver.org) for details.
+
+The version number for packages must convey meaningful information to customers. When a package's major version number is updated, it can be expected that there are changes that may break existing projects. A minor version update implies addition of non-breaking features.  
+
+## Support all application stages
+MRTK3 is designed to be used by developers of new and existing Unity projects. Features should not be added that limit the abilities of developers to create their applications, regardless of the current stage of development.
+
+## Centralized project authority
+MRTK3 is governed by a steering committee. This committee is comprised of members from multiple companies with a vested interest in the success of MRTK. Decisions are made per the documented [governance](GOVERNANCE.md).
+
+Being an open source project does not mean that everyone has the ability to approve or reject changes. The [maintainers group](MAINTAINERS.md) has the responsibility for determining if a change is appropriate and meets the project guidelines and standards. This is not to imply that the MRTK3 project does not wish to receive code review feedback from non-maintainers, we openly encourage feedback and will take it under advisement when making approval decisions. 
+
+## Get community feedback on large decisions
+The community are our partners and they will have the opporunity to voice their opinions on large decisions. This feedback will be taken into account by the steering committee and weighed against the "needs of the many".
+
+Examples of decisions in which the community feedback has been solicited include; minimum Unity version for MRTK3 GA, selection of Unity XR Interaction Toolkit as the underlying abstraction layer, etc.
+
+## Avoid backchannel code reviews
+Code review feedback benefits everybody. All feedback should be documented in the pull request so that everyone can learn from the discussions and understand the criteria by which the change was approved. In person conversations are not discouraged, provided that the feedback is documented in the appropriate pull request.
+
+## Code owners
+MRTK3 uses a list of per-package code owners. This is to enable a default set of code reviewers to be assigned for each pull request. By default, the [maintainers group](MAINTAINERS.md) is considered owners for each package. Other individuals may also be added as appropriate.
```

Comment:
I think we should incorporate this information into the existing contributions documentation.   For example, there is already a code owners section at https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/blob/main/contributions/code-owners.md

Can you refactor this documentation to be part of the documents under https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/blob/main/contributions/

Reply:
removed code owners section
moved file to contributions folder and renamed to project-tenets.md